.travis.yml

```diff
@@ -12,12 +12,12 @@ env:
   - GEVENT=false
 
 install:
-  - pip install gevent PyYaml
+  - pip install gevent PyYaml pytest-random-order
   - pip install sphinx==1.7.9 ghp-import
 
 script:
-  - python -m pytest -vv tests
-  - python -m pytest -vv --doctest-modules easypy
+  - python -m pytest --random-order --random-order-bucket=package -vv tests
```

Comment:
python -m pytest --random-order --random-order-bucket=package -vv tests
usage: pytest.py [options] [file_or_dir] [file_or_dir] [...]
pytest.py: error: ambiguous option: --random-order could match --random-order-bucket, --random-order-seed
The command "python -m pytest --random-order --random-order-bucket=package -vv tests" exited with 4.

From Travis' output (happens only on Python 3.4).